cpu/esp32/include/sdkconfig_esp32c3.h

```diff
@@ -94,6 +94,11 @@ extern "C" {
 #define CONFIG_ESP_SLEEP_POWER_DOWN_FLASH       1
 #define CONFIG_ESP_SLEEP_GPIO_RESET_WORKAROUND  1
 
+/**
+ * ESP32-S3 specific USB configuration
```

Comment:
```suggestion
 * ESP32-C3 specific USB configuration
```

Copy-paste error?

Reply:
:sunglasses: good catch, thanks. I fixed and squashed it directly.